src/archivematicaCommon/lib/bindpid.py

```diff
@@ -0,0 +1,459 @@
+                        ' (ini-style) that specifies values for any of the'
+                        ' following arguments: [{}]'.format(
+                            '], ['.join(x.upper() for x in CFGABLE_PARAMS)))
+    # Resolve URL templates: these are Django/Jinja2 templates that are used to
+    # construct the URLs that PIDs/PURLs will resolve to.
+    parser.add_argument('--resolve-url-template-archive', type=str,
+                        action='store', dest='resolve_url_template_archive',
+                        help="Template for the URL that a DIP's PURL should"
+                        " resolve to", default=None)
+    parser.add_argument('--resolve-url-template-mets', type=str, action='store',
+                        dest='resolve_url_template_mets', help="Template for"
+                        " the URL that a DIP's PURL with the \"mets\" qualifier"
+                        " should resolve to.", default=None)
+    parser.add_argument('--resolve-url-template-file', type=str, action='store',
+                        dest='resolve_url_template_file', help="Template for"
+                        " the URL that a file's PURL should resolve to (Note:"
+                        " defaults to access)", default=None)
+    parser.add_argument('--resolve-url-template-file-access', type=str,
+                        action='store', dest='resolve_url_template_file_access',
+                        help="Template for the URL that a file's PURL with the"
+                        " \"access\" qualifier should resolve to", default=None)
+    parser.add_argument('--resolve-url-template-file-preservation', type=str,
+                        action='store',
+                        dest='resolve_url_template_file_preservation',
+                        help="Template for the URL that a file's PURL with the"
+                        " \"preservation\" qualifier should resolve to",
+                        default=None)
+    parser.add_argument('--resolve-url-template-file-original', type=str,
+                        action='store',
+                        dest='resolve_url_template_file_original',
+                        help="Template for the URL that a file's PURL with the"
+                        " \"original\" qualifier should resolve to",
+                        default=None)
+    return parser, parser.parse_args()
+
+
+def _parse_config(args):
+    """Parse the Handle config file whose path is referenced in
+    ``args.config_file``, if there is such a reference. Return a dict of config
+    attributes and values or an empty dict if there is no config.
+    """
+    cf = args.config_file
+    if not cf:
+        return {}
+    if not os.path.isfile(cf):
+        print('Warning: there is no config file at {}'.format(cf))
+        return {}
+    config = configparser.ConfigParser()
+    with open(cf) as filei:
+        config.read_file(filei)
+    return {key: config.get('Handle', key, fallback=None) for key in CFGABLE_PARAMS}
+
```

Comment:
Good catch. On my dev machine I must have installed `configparser` in my 2.7.6 Python so I wasn't seeing this error. With the changes coming in, I have tested command-line usage of bindpid.py and it seems to be working on 2.7.6, 2.7.8 and 3.4.2.